runtime/src/accounts_db.rs

```diff
@@ -3123,6 +3153,7 @@ impl AccountsDB {
                     Some(Box::new(move |_, _| flushed_store.clone())),
                     None,
                     false,
+                    false,
```

Comment:
nits: how about defining another thin wrapper for humans like `store_accounts_by_replacing`, which just forwards args by appending `false, false` at last:

```rust
fn store_accounts_by_replacing<'a>(
    &'a self,
    slot: Slot,
    accounts: &[(&Pubkey, &Account)],
    hashes: &[Hash],
    storage_finder: Option<StorageFinder<'a>>,
    write_version_producer: Option<Box<dyn Iterator<Item = u64>>>,
) -> StoreAccountsTiming {
    self.store_accounts_custom(..., false, false)
}
```

This is because:
- hard to grasp with increasing number of arguments.
- similar to this(https://github.com/solana-labs/solana/pull/15022/files#r569329203), let's make intended code-path-dependent assumptions explicit by naming things clearly.
- it seems that all of callers currently passing `false, false` indicating this is common arguments
- imo, `store_accounts**_by_replacing**` should be decent/helpful name to be called from the flushing and shrinking (v1/v2) paths. or `store_existing_accounts_again()`?
- lastly, I think `{false,true}, /* foo_bar */` is a good sign to refactor code itself.... ;)

Reply:
Yes. Did this.